lib/project_types/script/messages/messages.rb

```diff
@@ -103,9 +103,9 @@ module Messages
           metadata_schema_versions_missing_minor: "Invalid Script API metadata:" \
                                                   " 'schemaVersions' is missing the 'minor' field",
 
-          metadata_not_found_cause: "Can't find the script version file (%s).",
+          metadata_not_found_cause: "Can't find the script version file (%{filename}).",
           metadata_not_found_help: "Make sure your project is up-to-date and a script metadata file n" \
```

Comment:
I know this isn't your change but this string has a trailing `n`.
Results in weird sentence: `and a script metadata file nis accessible at`